coverage/.resultset.json

```diff
@@ -1,18 +1,483 @@
 {
   "RSpec": {
     "coverage": {
-      "/Users/Vivian_Chu/git/dmr/app/helpers/application_helper.rb": [
+      "/Users/Vivian_Chu/git/dmr/app/controllers/application_controller.rb": [
```

Comment:
do these files need to be in version control? If not, can you delete them in this PR?